usaspending_api/etl/submission_loader_helpers/object_class.py

```diff
@@ -41,20 +41,19 @@ def get_object_class_row(row):
     # Alias this to cut down on line lengths a little below.
     ocdr = ObjectClass.DIRECT_REIMBURSABLE
 
-    if len(object_class) == 4:
-        # this is a 4 digit object class, 1st digit = direct/reimbursable information
-        direct_reimbursable = ocdr.LEADING_DIGIT_MAPPING[object_class[0]]
-        object_class = object_class[1:]
-    else:
-        # the object class field is the 3 digit version, so grab direct/reimbursable information from a separate field
-        try:
-            direct_reimbursable = ocdr.BY_DIRECT_REIMBURSABLE_FUN_MAPPING[row.by_direct_reimbursable_fun]
-        except KeyError:
-            # So Broker sort of validates this data, but not really.  It warns submitters that their data
-            # is bad but doesn't force them to actually fix it.  As such, we are going to just ignore
-            # anything we do not recognize.  Terrible solution, but it's what we've been doing to date
-            # and I don't have a better one.
-            direct_reimbursable = None
+    if len(object_class) == 4 and object_class[3] == "0":
```

Comment:
Should any additional handling be put in place in case there is an Object Class of length 4 that doesn't end with "0"? Right now if that case ever happened to occur it seems it would make it through without an issue.

Reply:
That makes sense. I can account for that.